src/pages/Detail/DetailHeader/Components/DetailHeaderWannaWatchingModal/DetailHeaderWannaWatchingModal.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import WannaWatchingModalMovieDetail from '../WannaWatchingModalMovieDetail/WannaWatchingModalMovieDetail';
+import WannaWatchingBTN from '../WannaWatchingBTN/WannaWatchingBTN';
+import BeingWatchingBTN from '../BeingWatchingBTN/BeingWatchingBTN';
+
+import DontCareBTN from '../DontCareBTN/DontCareBTN';
+import './DetailHeaderWannaWatchingModal.scss';
+
+class DetailHeaderWannaWatchingModal extends React.Component {
+  render() {
+    const {
+      isDefaultButton,
+      isActiveBTN,
+      isclickedWannaWatchingBTN,
+      isclickedBeingWatchingBTN,
+      handleResetButtonStatus,
+      handleActiveBeingWatchingBTN,
+      handleActiveWannaWatchingBTN,
+      onWannaWatchingModalToggle,
+      dontCareMovie,
+      handleControlDontCareButton,
+    } = this.props;
+    return (
+      <section className="DetailHeaderModal">
+        <div className="DetailHeaderModalBackGround">
+          <WannaWatchingModalMovieDetail />
+          <div className="DetailHeaderModalBTNs">
+            <WannaWatchingBTN
+              isDefaultButton={isDefaultButton}
+              isActiveBTN={isActiveBTN}
+              isclickedWannaWatchingBTN={isclickedWannaWatchingBTN}
+              isclickedBeingWatchingBTN={isclickedBeingWatchingBTN}
+              handleResetButtonStatus={handleResetButtonStatus}
+              handleActiveWannaWatchingBTN={handleActiveWannaWatchingBTN}
+            />
+            <BeingWatchingBTN
+              isDefaultButton={isDefaultButton}
+              isActiveBTN={isActiveBTN}
+              isclickedWannaWatchingBTN={isclickedWannaWatchingBTN}
+              isclickedBeingWatchingBTN={isclickedBeingWatchingBTN}
+              handleActiveBeingWatchingBTN={handleActiveBeingWatchingBTN}
+              handleResetButtonStatus={handleResetButtonStatus}
+              handleControlDontCareButton={handleControlDontCareButton}
+            />
+          </div>
+
+          <DontCareBTN
+            isDefaultButton={isDefaultButton}
+            isActiveBTN={isActiveBTN}
+            isclickedWannaWatchingBTN={isclickedWannaWatchingBTN}
+            isclickedBeingWatchingBTN={isclickedBeingWatchingBTN}
+            dontCareMovie={dontCareMovie}
+            handleResetButtonStatus={handleResetButtonStatus}
+            handleControlDontCareButton={handleControlDontCareButton}
+          />
```

Comment:
동일한 props를 여러개의 컴포넌트에 전달할때 스프레드 시트를 사용할 수 있습니다
```suggestion
const passingProps = {
      isDefaultButton: isDefaultButton,
      isActiveBTN: isActiveBTN,
      isclickedWannaWatchingBTN: isclickedWannaWatchingBTN,
      isclickedBeingWatchingBTN: isclickedBeingWatchingBTN,
      handleResetButtonStatus: handleResetButtonStatus,
      handleActiveWannaWatchingBTN: handleActiveWannaWatchingBTN,
    };

    return (
      <section className="DetailHeaderModal">
        <div className="DetailHeaderModalBackGround">
          <WannaWatchingModalMovieDetail />
          <div className="DetailHeaderModalBTNs">
            <WannaWatchingBTN {...passingProps} />
            <BeingWatchingBTN
              {...passingProps}
              handleControlDontCareButton={handleControlDontCareButton}
            />
          </div>

          <DontCareBTN
            {...passingProps}
            dontCareMovie={dontCareMovie}
            handleControlDontCareButton={handleControlDontCareButton}
          />
```

Reply:
이방법도 업청 획기적이네요.. 정말 잘 배워갑니다.